### ISSUE TITLE
How to store pip under /usr/local/lib

### ISSUE THREAD

**Issue by avoutsas67** [pypa/get-pip#167]
Hello, 

When running get-pip.py pip in Ubuntu 18.04 pip installed under .local/bin and .local/lib. Is there a way to install pip under /usr/local/bin and /usr/local/lib? 

Apologies that I raised an issue to ask a question. 

Many thanks for your support
Achilleas

**Comment by FrankAllotta** [comment]
I would also like to know how to do this as it would be nice to have pip available globally.

**Comment by Yi-CDC** [comment]
Yes, it's a good question. I want to install pip globally not only in personal's .local.

**Comment by edmorley** [comment]
@avoutsas67 Hi! get-pip.py doesn't control where the bin scripts for installed packages end up - that's something that will be controlled by your Python distributions configuration (in your case, presumably the Python provided by Ubuntu).

As such, I don't think your issue is related to this repository.
